Doc/library/importlib.rst

```diff
@@ -420,30 +420,34 @@ ABC hierarchy::
         reloaded):
 
         - :attr:`__name__`
-            The name of the module.
+            The module's fully-qualified name.
 
         - :attr:`__file__`
-            The path to where the module data is stored (not set for built-in
-            modules).
+            The location of the module's source code.
+            It is not set for built-in modules and frozen modules
+            and ``None`` for namespace packages.
```

Comment:
This reads just a bit awkwardly for me.  What do you think about "It is ``None`` for namespace packages and not set for built-in and frozen modules." ?

Reply:
Okay, updated.